vignettes/comparisons.Rmd

```diff
@@ -68,7 +68,7 @@ comparisons(mod,
 
 In GLM models, most quantities of interest are *conditional*, in the sense that they will typically depend on the values of all the predictors in the model. Therefore, we need to decide *where* in the predictor space we want to evaluate the quantity of interest described above.
 
-By default, `comparisons()` will compute estimates for every row of the original dataset that was used to fit a model. There are `r nrow(dat)` observations in the titanic dataset. Therefore, if we just execute the code in the previous section, we will obtain `r nrow(dat)` estimates of the difference between the probability of survival in 3rd and 1st class:
+By default, `comparisons()` will compute estimates for every row of the original dataset that was used to fit a model. The estimation was made on `r nrow(mod$model)` observations in the titanic dataset. Therefore, if we just execute the code in the previous section, we will obtain `r nrow(mod$model)` estimates of the difference between the probability of survival in 3rd and 1st class:
```

Comment:
557 observations are dropped before the estimation due to missing values so there was an inconsistency between the number of obs in the text and the output of `comparisons()`

Reply:
Thanks!